### PR TITLE
ADEN-8484 Log catlapsing info without adengine_debug=1 query param.

### PR DESCRIPTION
**Pull request by jacekbj**
## Links

* http://wikia-inc.atlassian.net/browse/JIRA-123
* more relevant links

## Description

A description written using complete sentences of the changes being made and any context necessary to understand them.

## Reviewers

Use `@` mentions here. Alternatively, assign the pull request to a person.

### CODE REVIEW

File: app/modules/ads/bill-the-lizard-wrapper.js [hunk 1]
```diff
@@ -109,6 +109,8 @@ export const billTheLizardWrapper = {
         slotService.disable(getNextIncontentId(), 'catlapsed');
 
         utils.logger(logGroup, `catlapsing ${getNextIncontentId()}`);
+        // eslint-disable-next-line no-console
+        console.log(`catlapsing ${getNextIncontentId()}`);
```
Comment (nandy-andy) [inline app/modules/ads/bill-the-lizard-wrapper.js:118]: We have a deal! If we use `catlapsing ${getNextIncontentId()}` in one more place we'll assign it to a variable which we'll reuse ;-)

**Review by nandy-andy — APPROVED** [review]

**Comment by fraszczakszymon** [comment]
😈idea: @jacekbj what do you think about possibility to enable adengine-logger through cookie/localStorage? So we can enable logger via `?adengine_debug=<1|logGroup>` **and** 
```
window.ads.enableDebugMode(logGroup = 1)
```
?

**Comment by jacekbj** [comment]
Great idea! For that we will need a separate ticket though.

File: config/bundlesize.js [hunk 2]
```diff
@@ -5,7 +5,7 @@ const assetsFolder = 'mobile-wiki/assets';
 module.exports = {
   'mobile-wiki.js': {
     pattern: `${assetsFolder}/mobile-wiki-*.js`,
-    limit: '457KB',
+    limit: '458KB',
```
Comment (dmnsobczak) [inline config/bundlesize.js]: 💎

File: app/modules/ads/bill-the-lizard-wrapper.js (hunk 1 above)
Comment (bkoval) [reply app/modules/ads/bill-the-lizard-wrapper.js:118]: Hi! Why would we want console.logs on production? :)

Comment (nandy-andy) [reply app/modules/ads/bill-the-lizard-wrapper.js:118]: Good question and my bad! I've checked how we do it in "a kosher way" and we can improve it most likely and do consistently: https://github.com/Wikia/mobile-wiki/blob/501663b1715a19d72d2b745a5308883e2dcbd354/app/components/ad-slot.js#L55

Comment (jacekbj) [reply app/modules/ads/bill-the-lizard-wrapper.js:118]: I had no idea such service exists, my bad :/

**Pull request merged**